src/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "nativescript-live-engage",
-	"version": "6.0.4",
+	"version": "6.0.5",
```

Comment:
Maybe to raise version to 6.1.0? It is bigger change since version of sdk is incremented.